packages/language-server/server/server.js

```diff
@@ -1,68 +1,32 @@
-const path = require("path");
-const babylon = require("@babel/parser");
-const { getConfig } = require("@hegel/cli/lib/config");
-const { importModule } = require("@hegel/cli/lib/module");
-const { promises: { readFile } } = require("fs");
 const {
-  createGlobalScope,
-  createModuleScope,
   PositionedModuleScope,
-} = require("@hegel/core");
+} = require("@hegel/core/type-graph/module-scope");
 const {
   TextDocuments,
   IPCMessageReader,
   IPCMessageWriter,
   createConnection,
-  DiagnosticSeverity
 } = require("vscode-languageserver");
+const { TextDocument } = require("vscode-languageserver-textdocument");
 
-let types = {},
-  errors = [],
-  stdLibTypeGraph,
-  nodeJsGlobalTypeGraph,
-  browserGlobalTypeGraph,
-  config;
-const documents = new TextDocuments();
+const { validateTextDocument, types } = require("./validation/code_validation");
```

Comment:
Seems like I need to write a contribution guide. So, sorry, that I make the remark.
But, could you please sort the libraries includes by the identificators length. For example:
```
const { TextDocument } = require("vscode-languageserver-textdocument");
const { convertRangeToLoc } = require("./utils/range");
const { validateTextDocument, types } = require("./validation/code_validation");
const {
  TextDocuments,
  IPCMessageReader,
  IPCMessageWriter,
  createConnection,
  DiagnosticSeverity
} = require("vscode-languageserver");

```